ironic-deployment/README.md

```diff
@@ -4,6 +4,10 @@ This folder contains kustomizations for Ironic.
 They are mainly used through the [deploy.sh](../tools/deploy.sh) script, which takes care of generating the necessary config for basic-auth and TLS.
 
 - **base** - This is the kustomize base that we start from.
-- **components** - In here you will find re-usable kustomize components for running Ironic with TLS, basic-auth or keepalived. Note that the basic-auth component is missing the actual credentials. This is on purpose, to make sure that the user is setting the password. Similarly the TLS component needs to have the proper IP/SAN set for the certificates.
+- **components** - In here you will find re-usable kustomize components for running Ironic with TLS, basic-auth, keepalived or mariadb.
+   - **basic-auth** - Enable basic authentication. Note that the basic-auth component is missing the actual credentials. This is on purpose, to make sure that the user is setting the password.
+   - **tls** - Enable TLS. The TLS component needs to have the proper IP/SAN set for the certificates.
+   - **keepalived** - Add a keepalived container to the deployment. This is useful when using a VIP for exposing Ironic, so that the IP can move with the pod.
```

Comment:
```suggestion
   - **keepalived** - Add a keepalived container to the deployment. This is useful when using a VIP for exposing Ironic endpoint, so that the IP can move with the pod.
```